packages/web/src/pages/saved-page/components/emptyStateMessages.ts

```diff
@@ -0,0 +1,18 @@
+export const emptyStateMessages = {
```

Comment:
Is there a reason emptyStateMessages is separate for desktop but defined in the messages object for mobile? Not a big deal but we could potentially be more consistent here

Reply:
yeah it's because in web it's shared by mobile web and desktop components so it was getting super duplicative! can move to common if you think that's better, but i thought that might be overkill